test/old/testdir/test_normal.vim

```diff
@@ -3897,9 +3897,9 @@ func Test_normal_count_after_operator()
   bw!
 endfunc
 
-func Test_normal_gj_on_extra_wide_char()
+func Test_normal_gj_on_unprintable_char()
```

Comment:
"extra wide" refers to the unprintable char taking 6 cells. "unprintable" alone doesn't necessarily mean what.

Reply:
"extra wide" absolutely does not imply that something is at least 6 cells wide. 
"unprintable" refers to a documented, pre-existing thing which is the subject of this test.

if the exact width of 6 is important I will add a comment.